website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,271 @@
+or check the following table that gives some script outputs for you:
+
+| OVA name                                                 | TKR_VERSION               | TKGVERSION |
+|----------------------------------------------------------|---------------------------|------------|
+| v1.19.16+vmware.1-tkg.2-fba68db15591c15fcd5f26b512663a42 | v1.19.16---vmware.1-tkg.2 | v1.4.3     |
+| v1.20.14+vmware.1-tkg.2-5a5027ce2528a6229acb35b38ff8084e | v1.20.14---vmware.1-tkg.2 | v1.4.3     |
+| v1.20.15+vmware.1-tkg.2-839faf7d1fa7fa356be22b72170ce1a8 | v1.20.15---vmware.1-tkg.2 | v1.5.4     |
+| v1.21.8+vmware.1-tkg.2-ed3c93616a02968be452fe1934a1d37c  | v1.21.8---vmware.1-tkg.2  | v1.4.3     |
+| v1.21.11+vmware.1-tkg.2-d788dbbb335710c0a0d1a28670057896 | v1.21.11---vmware.1-tkg.2 | v1.5.4     |
+| v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933  | v1.22.9---vmware.1-tkg.1  | v1.5.4     |
+| v1.22.13+vmware.1-tkg.2-ea08b304658a6cf17f5e74dc0ab7544f | v1.22.13---vmware.1-tkg.1 | v1.6.1     |
+| v1.21.14+vmware.2-tkg.5-d793afae5aa18e50bd9175e339904496 | v1.21.14---vmware.2-tkg.5 | v1.6.1     |
+| v1.23.10+vmware.1-tkg.2-b53d41690f8742e7388f2c553fd9a181 | v1.23.10---vmware.1-tkg.1 | v1.6.1     |
+
+In [the TKGm cluster creation example][cluster], the built-in Terraform function `templatefile` is used to substitute every placeholder
+mentioned above with its final value. The returned value is the CAPVCD YAML payload that needs to be set in the `capi_yaml` placeholder in the
+JSON template.
+
+~> Notice that we need to replace `\n` with `\\n` and also `\"` to `\\\"` to avoid breaking the JSON contents when the YAML is set, as
+line breaks and double quotes would not be interpreted correctly otherwise. This is also done in [the mentioned example][cluster].
+
+When you have set all the required values, a `terraform apply` should trigger a TKGm cluster creation. The operation is asynchronous, meaning that
+you need to monitor the RDE `computed_entity` value to see the status of the cluster provisioning. Some interesting output examples:
+
+```hcl
+locals {
+  k8s_cluster_computed = jsondecode(vcd_rde.k8s_cluster_instance.computed_entity)
+  has_status           = lookup(local.k8s_cluster_computed, "status", null) != null
+}
+
+# Outputs the TKGm Cluster creation status
+output "computed_k8s_cluster_status" {
+  value = local.has_status ? local.k8s_cluster_computed["status"]["vcdKe"]["state"] : null
+}
+
+# Outputs the TKGm Cluster creation events
+output "computed_k8s_cluster_events" {
+  value = local.has_status ? local.k8s_cluster_computed["status"]["vcdKe"]["eventSet"] : null
+}
+
+```
+
+When the status displayed by `computed_k8s_cluster_status` is `provisioned`, it will mean that the TKGm cluster is successfully provisioned and
+the Kubeconfig is available and ready to use. You can retrieve it with:
```

Comment:
```suggestion
When the status displayed by `computed_k8s_cluster_status` is `provisioned`, it will mean that the TKGm cluster is successfully provisioned and
the Kubeconfig is available and ready to use. It can be retrieved with:
```

Reply:
Fixed!